tests/test_validators.py

```diff
@@ -69,7 +69,9 @@ def test_repr(self):
         expected = dedent("""
             UniquenessSerializer():
                 id = IntegerField(label='ID', read_only=True)
-                username = CharField(max_length=100, validators=[<UniqueValidator(queryset=UniquenessModel.objects.all())>])
```

Comment:
Don't change this string - it's used in a literal string comparison, and changing the formatting will break the test.